pydash/arrays.py

```diff
@@ -1813,6 +1912,9 @@ def iterunique(array, comparator=None, iteratee=None):
             seen.append(cmp_item)
 
 
+iterunion = iterunique
```

Comment:
Well using `iterunique` method for a `union` method makes me confusing. I would think, how come "they are using unique method inside union methods". You can say the functionality is the same and why a different name. But, imagine a person coming and seeing the code - he/she would think the same as above.